src/Microsoft.Health.Extensions.Xunit/Microsoft.Health.Extensions.Xunit.csproj

```diff
@@ -8,6 +8,10 @@
   <ItemGroup>
     <PackageReference Include="Ensure.That" Version="10.1.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.MSBuild" Version="2.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
What is this change for?

Reply:
Not sure what that is from. I'll remove it.

> What is this change for?

Not sure what that is from. I'll remove it.